src/Template/SentencesLists/show_angular.ctp

```diff
@@ -186,37 +187,37 @@ $this->set('title_for_layout', $this->Pages->formatTitle($listName));
 
     <md-content ng-cloak>
     <?php
-    if ($permissions['canAddSentences'] && count($sentencesInList) == 0) {
+    if (count($sentencesInList) == 0) {
         ?>
         <div class="no-sentence-info" ng-if="!vm.showForm && vm.sentences.length === 0">
-            <p><?= __('This list is empty.') ?></p>
-            <div class="hint">
+            <p>
                 <?php
-                echo format(
-                    __(
-                        'You can create new sentences directly in this list by clicking on the '.
-                        '{addSentenceButton} icon in the header of this section.', true
-                    ),
-                    ['addSentenceButton' => '<md-icon>add</md-icon>']
-                );
+                    echo format(
+                        __(
+                            $filterLanguage=='und'?'This list is empty.':'This list does not contain any sentences in {language}'
+                        ),
+                        ['language'=> $this->Languages->codeToNameAlone($filterLanguage)]
+                    );
                 ?>
-            </div>
-            <div class="hint">
-                <?php
-                echo format(
-                    __(
-                        'You can also add existing sentences to this list, from other pages, by clicking on '.
-                        'the {addToListButton} icon in the menu of the sentence.', true
-                    ),
-                    ['addToListButton' => '<md-icon>list</md-icon>']
-                );
+            </p>
+            <?php
+            if ($permissions['canAddSentences']){
                 ?>
-            </div>
+                <div class="hint">
+                    <?php
+                    echo format(
+                        __(
+                            'Click on the {addNewSentence} icon above to add a new sentence to the list. You can also add existing sentences to this list, from other pages, by clicking on '.
```

Comment:
Note that most supported UI languages aren't actively maintained. Adding a new UI string for a new feature is okay, and it's unavoidable that it might not get translated for a long time, but I think it's best not to change existing UI strings unless there's something wrong with them, to protect existing translations from getting lost.

So I'd keep the two hints separate and use the same wording as before.